app/src/main/java/com/example/root/smsreader/MainActivity.java

```diff
@@ -22,117 +15,116 @@
 
 public class MainActivity extends AppCompatActivity {
 
-public  int len_all_sms = 0;
-public int Count=0;
+    private static final String TAG = MainActivity.class.getSimpleName();
+    // Member variables have prefix "m". rest of the name uses CamelCase
+public  int mLenAllSms = 0;
+public int mCount = 0;
```

Comment:
The variables that you declare here are called **Member Variables** 
These are available throughout the class across all methods in it i.e. in **Member Methods**.

#### As a coding convention, while naming them, a prefix "m" is used.
So, `count` becomes `mCount`.

#### While naming variables or methods, use camelCase. 
This makes the variable more readable. 
Check for yourself.

`int isthisreadbleornot` or `int isThisReadableOrNot`